test/functionalTest/cases/787_cprs_full_functional/update_false_registration.test

```diff
@@ -0,0 +1,140 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Update With a False Registration
+
+--SHELL-INIT--
+dbInit CB
+dbInit CP1
+brokerStart CB
+brokerStart CP1
+
+--SHELL--
+
+#
+# 01. Register E1/T1/A1-CP1 in CB
+# 02. Update/UPDATE E1/T1/A1 in CB
+#
+
+echo "01. Register E1/T1/A1-CP1 in CB"
+echo "==============================="
+payload='{
+  "contextRegistrations": [
+    {
+      "entities": [
+        {
+           "type": "T1",
+           "isPattern": "false",
+           "id": "E1"
+        }
+      ],
+      "attributes": [
+        {
+          "name": "A1",
+          "type": "string",
+          "isDomain": "false"
+        }
+      ],
+      "providingApplication": "http://localhost:'${CP1_PORT}'/v1"
+    }
+ ],
+ "duration": "P1M"
+}'
+orionCurl --url /v1/registry/registerContext --payload "$payload" --json
+echo
+echo
+
+
+echo "02. Update/UPDATE E1/T1/A1 in CB"
+echo "================================"
+payload='{
+  "contextElements": [
+    {
+      "type": "T1",
+      "id":   "E1",
+      "attributes": [
+        {
+          "name": "A1",
+          "type": "string",
+          "value": "Via CB"
+        }
+      ]
+    }
+  ],
+  "updateAction": "UPDATE"
+}'
+orionCurl --url /v1/updateContext --payload "$payload" --json
+echo
+echo
+
+
+--REGEXPECT--
+01. Register E1/T1/A1-CP1 in CB
+===============================
+HTTP/1.1 200 OK
+Content-Length: 74
+Content-Type: application/json
+Date: REGEX(.*)
+
+{
+    "duration": "P1M", 
+    "registrationId": "REGEX([0-9a-f]{24})"
+}
+
+
+02. Update/UPDATE E1/T1/A1 in CB
```

Comment:
I understand that in this case CB tries to update forward to CPr but, given that CPr doesn't exist, the CB gets somekind of connection error and returns a 404. Is that correct?